xds/src/test/java/io/grpc/xds/ClusterResolverLoadBalancerTest.java

```diff
@@ -1166,15 +1166,5 @@ public void shutdown() {
       shutdown = true;
       childBalancers.remove(this);
     }
-
-    void deliverSubchannelState(final Subchannel subchannel, ConnectivityState state) {
```

Comment:
I've seen this one before, and it is a bit interesting. This FakeLoadBalancer is copied lots of places, and some places this method is used. We should really probably move it out of individual classes, and modifying it may make that harder.

Note, there's also fall-out from this change, as `helper` would be unused, so it'd need to be removed as a field, and then the constructor would change, and then all the callers have to change...